[SVG2] Add `fetchPriority` property support for `SVGImageElement`, `SVGFEImageElement` and `SVGScriptElement`

Ahmad-S792 opened the pull request:
#### f1e771f6cb988f0e92c9aaff580c4fc9a00416c8
<pre>
[SVG2] Add `fetchPriority` property support for `SVGImageElement`, `SVGFEImageElement` and `SVGScriptElement`
<a href="https://bugs.webkit.org/show_bug.cgi?id=293585">https://bugs.webkit.org/show_bug.cgi?id=293585</a>
<a href="https://rdar.apple.com/152044271">rdar://152044271</a>

Reviewed by NOBODY (OOPS!).

This patch aligns WebKit with Gecko / Firefox.

This patch aims to add `fetchPriority` to to following interfaces:

- SVGImageElement
- SVGFEImageElement
- SVGScriptElement

Basically, we are extending `fetchPriority` from HTML counter-part to
SVG.

* Source/WebCore/svg/SVGFEImageElement.cpp:
(WebCore::SVGFEImageElement::fetchPriorityForBindings const):
(WebCore::SVGFEImageElement::fetchPriority const):
* Source/WebCore/svg/SVGFEImageElement.h:
* Source/WebCore/svg/SVGFEImageElement.idl:
* Source/WebCore/svg/SVGImageElement.cpp:
(WebCore::SVGImageElement::fetchPriorityForBindings const):
(WebCore::SVGImageElement::fetchPriority const):
* Source/WebCore/svg/SVGImageElement.h:
* Source/WebCore/svg/SVGImageElement.idl:
* Source/WebCore/svg/SVGScriptElement.cpp:
(WebCore::SVGScriptElement::fetchPriorityForBindings const):
(WebCore::SVGScriptElement::fetchPriority const):
* Source/WebCore/svg/SVGScriptElement.h:
* Source/WebCore/svg/SVGScriptElement.idl:
* Source/WebCore/svg/svgattrs.in:
* LayoutTests/imported/w3c/web-platform-tests/svg/embedded/attr-image-fetchpriority-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/scripted/attr-script-fetchpriority-expected.txt:
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f1e771f6cb988f0e92c9aaff580c4fc9a00416c8

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/120688 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/40382 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/31034 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/127081 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/72763 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/3316b713-41c2-422c-b4bd-776b1f672558) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/122564 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/41079 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/48959 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/91670 "Passed tests") | [💥 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/60922 "An unexpected error occured. Recent messages:Printed configuration") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/99fa357c-a723-4d55-95bc-09b940bacf2a) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/123640 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/32807 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/108186 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/72220 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/641af175-1ab1-42d2-ab14-8d977d3eab0b) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/31836 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/26293 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/70686 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/102295 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/26475 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/129949 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/47609 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/36151 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/100292 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/47977 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/104368 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/100131 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/45579 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/23609 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/44269 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/47471 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/53176 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/46940 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/50286 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/48626 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

annevk reviewed (changes requested):
This does not seem to pass the value through to the networking code.

Ahmad-S792 committed:
[SVG2] Add `fetchPriority` property support for `SVGImageElement`, `SVGFEImageElement` and `SVGScriptElement`
https://bugs.webkit.org/show_bug.cgi?id=293585
rdar://152044271

Reviewed by NOBODY (OOPS!).

This patch aligns WebKit with Gecko / Firefox.

This patch aims to add `fetchPriority` to to following interfaces:

- SVGImageElement
- SVGFEImageElement
- SVGScriptElement

Basically, we are extending `fetchPriority` from HTML counter-part to
SVG.

* Source/WebCore/svg/SVGFEImageElement.cpp:
(WebCore::SVGFEImageElement::fetchPriorityForBindings const):
(WebCore::SVGFEImageElement::fetchPriority const):
* Source/WebCore/svg/SVGFEImageElement.h:
* Source/WebCore/svg/SVGFEImageElement.idl:
* Source/WebCore/svg/SVGImageElement.cpp:
(WebCore::SVGImageElement::fetchPriorityForBindings const):
(WebCore::SVGImageElement::fetchPriority const):
* Source/WebCore/svg/SVGImageElement.h:
* Source/WebCore/svg/SVGImageElement.idl:
* Source/WebCore/svg/SVGScriptElement.cpp:
(WebCore::SVGScriptElement::fetchPriorityForBindings const):
(WebCore::SVGScriptElement::fetchPriority const):
* Source/WebCore/svg/SVGScriptElement.h:
* Source/WebCore/svg/SVGScriptElement.idl:
* Source/WebCore/svg/svgattrs.in:
* LayoutTests/imported/w3c/web-platform-tests/svg/embedded/attr-image-fetchpriority-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/scripted/attr-script-fetchpriority-expected.txt:
